components/brave_welcome_ui/components/help-improve/index.tsx

```diff
@@ -65,6 +66,10 @@ function HelpImprove() {
     return WelcomeBrowserProxyImpl.getInstance().getWelcomeCompleteURL()
   })
 
+  // Show toggles only if the preference is not managed by policy
+  const [showStatsToggle] = React.useState(() => !loadTimeData.getBoolean('isStatsReportingEnabledManaged'))
+  const [showP3AToggle] = React.useState(() => !loadTimeData.getBoolean('isP3AEnabledManaged'))
```

Comment:
I'm not sure, but do loadTimeData need to be useState() hook? I guess it's not changed during life cycle of this component so maybe not worth using hook?

@fallaciousreasoning what do you think?

Reply:
I just followed the same pattern that `completeURLPromise` is using in this component, which also does not change during the lifetime of the component. I liked the lazy initialization that `useState` provides.

Reply:
This definitely doesn't seem like it should be wrapped in a `useState` - we can store it in a variable (either outside the component or inside it).

If you really want the lazy initialization, I think `useMemo` would be more idiomatic but I don't know whether its worth it for `getBoolean`

Reply:
1a19ff3b94a